tutorials/ai-core-azure/ai-core-azure.md

```diff
@@ -0,0 +1,79 @@
+---
+author_name: Dhrubajyoti Paul
+author_profile: https://github.com/dhrubpaul
+title: Setup Azure Bucket With AI Core
+description: In this tutorial we will be walking through the steps on how to Set up azure bucket (Azure Blob Storage) with SAP AI Core
+keywords: Azure Bucket setup
+auto_validation: true
+time: 10
+tags: [ tutorial>beginner, topic>artificial-intelligence, topic>machine-learning, software-product>sap-business-technology-platform, software-product>sap-ai-launchpad, software-product>sap-ai-core ]
+primary_tag: software-product>sap-ai-core
+parser: v2
+---
+# Setup Azure DataBucket With AI Core
```

Comment:
You do not need the title and description both here and in the meta data (rows 4 and 5 above).  Also, the title here is different than the title in row 4.  Here you say "DataBucket" but above, you just say "Bucket."  I recommend removing it from above and keeping this one.  Finally, "With" should be lower case as it is a preposition which does not need to be capitalized.